### DISCUSSION TITLE
Access violation on add skia units in Windows 64

### DISCUSSION THREAD

**Discussion by zLuckr0x** [skia4delphi/skia4delphi#177] [Q&A]
Access violation on add skia units in Windows 64

Im using FMX Platform and Target Windows 64, im add 2 units skia, skia.FMX and run, return me Access Violation, but run in 32 bits opened fine.

**Comment by viniciusfbb** [comment]
Hi! This occurs when you don't [enable your project to use Skia](https://github.com/skia4delphi/skia4delphi#enable-skia).

**Comment by zLuckr0x** [comment]
![Sem título](https://user-images.githubusercontent.com/31612412/200938979-fbeffbc6-d6e4-4c67-8116-6034f5e7dc08.png)

Im Enabled Skia, and now other error return for me, on run project.

**Reply by viniciusfbb** [reply]
I need more information, such as the version you are using.

Another thing, after enabling you must clean and recompile the project.

**Reply by viniciusfbb** [reply]
@zLuckr0x To better explain how it works: to use the library you need the skia dll on Windows and made some configuration in the project deployment for other platforms. To make everything easier, we made this plugin that when you [enable Skia](https://github.com/skia4delphi/skia4delphi#enable-skia), all the necessary deployments are automatically configured in the project, and on Windows, during builds, the Skia dll is automatically copied to the same folder as your app.

**Reply by viniciusfbb** [reply]
Any news? Can I close?
